### PR TITLE
Add Lark filter

### PR DESCRIPTION
**Pull request by jagtalon**
😬

## Real
img_4511
img_4512
img_4513

## CSS
screen shot 2015-11-14 at 22 55 47
screen shot 2015-11-14 at 22 56 33
screen shot 2015-11-14 at 22 56 50

### CODE REVIEW

**Comment by una** [comment]
Thanks for the PR!

This seems a little too contrasted for Lark -- check out the test page. I think something like: `filter: contrast(.9);` might work a bit better.

**Comment by jagtalon** [comment]
@una adjusted the filter based on the tests. Should be ready for another review!

**Comment by una** [comment]
Awesome, thanks!

File: source/scss/lark.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/*
+ * Lark
+ *
+ */
+@import 'shared';
+
+%lark,
+.lark {
+  @extend %filter-base;
+  filter: contrast(0.9);
```
Comment (una) [inline source/scss/lark.scss]: remove leading 0's please :)

File: source/scss/lark.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  &:before {
+    background: #22253f;
+    mix-blend-mode: color-dodge;
+  }
```
Comment (una) [inline source/scss/lark.scss]: I tested this out to cut some of the highlights:

```
&:after {
  background: rgba(242, 242, 242, .8);
  mix-blend-mode: darken;
}

File: source/scss/lark.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ */
+@import 'shared';
+
+%lark,
+.lark {
+  @extend %filter-base;
+  filter: contrast(0.9);
+
+  &:before {
+    background: #22253f;
```
Comment (una) [inline source/scss/lark.scss]: As per #97 -- we're going to use consistency via color names in RGB from now on. Please convert this hex code

**Comment by jagtalon** [comment]
@una thank you for the comments and that awesome blend mode in `:after`. Ready!

**Comment by una** [comment]
:+1: :tada:

**Pull request merged**